src/institute/models.py

```
@@ -139,5 +145,9 @@ def to_dict(self) -> Dict[str, Any]:
             "user_id": self.user_id,
             "role": self.role,
             "permissions": self.permissions,
-            "joined_at": self.joined_at
-        }
\ No newline at end of file
+            "workspace_type": self.workspace_type,
+            "workspace_name": self.workspace_name,
+            "joined_at": self.joined_at,
+        }
+        if self.class_id:
+            base_dict["class_id"] = self.class_id
```

Ruff (0.12.2)

Undefined name `base_dict`

(F821)